Components/GridPostsLatest/_style.scss

```diff
@@ -60,9 +63,15 @@
 
     &-inner {
       @include reset-content-margins;
+
+      flex-grow: 1;
     }
 
     &-link {
+      display: flex;
+      flex-direction: column;
+      flex-grow: 1;
+      justify-content: space-between;
```

Comment:
Do we need this line, as we don't want to have space anyways?

Reply:
You are right, the line is un-necessary and will be removed before merging.